internal/transport/transport_test.go

```diff
@@ -780,10 +780,18 @@ func (s) TestGracefulClose(t *testing.T) {
 		go func() {
 			defer wg.Done()
 			str, err := ct.NewStream(ctx, &CallHdr{})
-			if err == ErrConnClosing {
-				return
-			} else if err != nil {
-				t.Errorf("_.NewStream(_, _) = _, %v, want _, %v", err, ErrConnClosing)
+			if err != nil {
+				// The NewStream calls race with closing of the transport (after
+				// the last stream is closed).
+				// - if it is after the transport is closed (case <-ct.ctxDone),
+				//   we don't care about the error.
```

Comment:
Why would the transport be closed? It was _gracefully_ closed, but we have an active stream (from line 755) so it should never close, should it?  Unless the server ends the stream, but it looks like it does not, until it receives the client's end-stream.

Reply:
We start 200 goroutines to run this.
Some of them could be so slow that they run after the end of the line 755 stream.